packages/ds-medicare-gov/src/styles/components/_Dialog.scss

```diff
@@ -9,7 +9,6 @@
 
     .ds-c-icon {
       color: var(--color-gray-dark);
-      font-size: 22px;
```

Comment:
This style can be removed as dialogs now have a token controlling close icon sizing in each theme.